models/sipnet/R/write_restart.SIPNET.R

```diff
@@ -55,11 +58,17 @@ write_restart.SIPNET <- function(outdir, runid, start.time, stop.time, settings,
     analysis.save[[length(analysis.save) + 1]] <- udunits2::ud.convert(new.state$NPP, "kg/m^2/s", "Mg/ha/yr")  #*unit.conv -> Mg/ha/yr
     names(analysis.save[[length(analysis.save)]]) <- c("NPP")
   }
+
+  if ("NEE" %in% variables) {
+    analysis.save[[length(analysis.save) + 1]] <- new.state$NEE
+    if (new.state$NEE < 0) analysis.save[[length(analysis.save)]] <- 0
+    names(analysis.save[[length(analysis.save)]]) <- c("NEE")
+  }
```

Comment:
Unclear why this code exits. NEE < 0 is perfectly valid, and indeed is the norm for most ecosystems (net carbon sinks) so I don't see why those values are being set to 0.

Reply:
I think I was adding NEE in because it was not already in there. It was copied from another variable and was not edit beyond changing the name to NEE. I've remove line 64

Reply:
Strange, I looked at the "Files Changed" tab and it shows this line as removed? But here on the "conversations" tab it still shows line 64 as changing NEE to be >=0?  GitHub bug?

Reply:
Still need to fix this line (line 64 should be deleted, comment suggested that it was, but it's still here)

Reply:
this was the thing Shawn was mentioning. If you look at the actual file in my branch, it is changed. but for some reason on here it is not. I Double checked my write_restart file, and it is changed, and I re-pushed the branch, just in case I didn't push it correctly or something.

Reply:
In fact, if you view the file, you'll see line 64 isn't in there anymore.